pkg/daemon/manager/manager.go

```diff
@@ -150,6 +151,37 @@ func (m *manager) EnsureServiceState(service winsvc.Service, state svc.State) er
 	return winsvc.WaitForState(service, state)
 }
 
+// Stop the service, and wait for the process associated with the service to stop
+func (m *manager) stopServiceAndProcess(winSvc *mgr.Service) error {
+	status, err := winSvc.Query()
+	if err != nil {
+		return errors.Wrap(err, "error querying service")
+	}
+	var pHandle windows.Handle
+	// A value of 0 indicates that no process is running
+	if status.ProcessId != 0 {
+		pHandle, err = windows.OpenProcess(windows.PROCESS_TERMINATE|windows.PROCESS_QUERY_INFORMATION, false,
+			status.ProcessId)
+		if err != nil {
```

Comment:
Could it ever be the case that the PID is non-zero but the process doesn't exist (timing issue)? If so, consider checking
``` golang
errors.Is(err, windows.ERROR_PROC_NOT_FOUND)
```

Reply:
My understanding is that ERROR_PROC_NOT_FOUND refers to a procedure, not a process, I wasn't able to find a specific error to handle here.

Reply:
As for if the case is possible, it would only be possible if the process crashed, or someone killed the process in the very small time between querying the service and trying to open the process.

Reply:
> I wasn't able to find a specific error to handle here

Me neither. I am OK with how things are written currently